charts/rancher-logging/rancher-logging/100.0.0+up3.10.0/templates/loggings/rke2/configmap.yaml

```diff
@@ -13,18 +13,42 @@ data:
         Daemon            Off
         Log_Level         info
         Coro_Stack_Size   24576
+        Parsers_File      parsers.conf
 
     [INPUT]
         Name              systemd
         Tag               rke2
-        Path              {{ .Values.systemdLogPath | default "/var/log/journal" }}
+        Path              {{ .Values.systemdLogPath | default "/var/run/journal" }}
```

Comment:
this was just to match what was in values.yaml, there should be no functional changes